modules/conversion-center/admin-menu-items/conversion-center-menu-item.php

```diff
@@ -0,0 +1,38 @@
+<?php
+namespace Elementor\Modules\ConversionCenter\AdminMenuItems;
+
+use Elementor\Core\Admin\Menu\Interfaces\Admin_Menu_Item;
+use Elementor\Core\Admin\Menu\Interfaces\Admin_Menu_Item_Has_Position;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+class Conversion_Center_Menu_Item implements Admin_Menu_Item, Admin_Menu_Item_Has_Position {
+
+	const POSITION = 58.7;
```

Comment:
```suggestion
	const AFTER_ELEMENTOR = 58.7;
```
@nicoladj77 What does this mean? Can it be more descriptive? Like why 58.7?